client/my-sites/plugins/plugin-details-sidebar/style.scss

```diff
@@ -10,6 +10,12 @@
 }
 
 .plugin-details-sidebar__plugin-details-content {
+	margin: 32px 0;
+	padding: 32px 0;
+	border-style: solid none;
+	border-width: 1px;
+	border-color: #eeeeee;
```

Comment:
Is there a variable we can use here?

Reply:
This variable has been used in some places in Calypso, none of them as a variable. I don't really know the reason.